Makefile

```diff
@@ -21,6 +21,15 @@ DATE    ?= $(shell git log -1 --format="%ct" | xargs -I{} date -d @{} +'%Y-%m-%d
 CNINFRA := go.ligato.io/cn-infra/v2/agent
 LDFLAGS = -X $(CNINFRA).BuildVersion=$(VERSION) -X $(CNINFRA).CommitHash=$(COMMIT) -X $(CNINFRA).BuildDate=$(DATE)
 
+RELEASE_TAG ?= $(shell git describe --always --tags --dirty --exact-match 2>/dev/null)
```

Comment:
The `--dirty` should be omitted in this case to avoid naming the release with `-dirty` suffix.

Reply:
Added the check to throw error if repo is dirty.